earth2mip/regrid.py

```diff
@@ -59,6 +59,11 @@ def forward(self, x):
         return x
 
 
+class DropLast(torch.nn.Module):
```

Comment:
Nit: DropLastLat? Or something to indicate latitude thats dropped here. Non-blocking

Reply:
Renamed to DropSouthPole